FINS/FINS-IOC-01App/Db/sans2d-vac-beamstop-inhibit.template

```diff
@@ -0,0 +1,17 @@
+record(calcout, "$(FULL_PREFIX)INHIBIT_FRONTBEAMSTOP") {
+    field(DESC, "Inhibit fbs movement if det rot > 0")
+    field(INPA, "$(P)MOT:FRONTDETROT CP")
+    field(INPB, "$(P)MOT:FRONTDETROT:SP CP")
+    field(CALC, "A > 0 || B > 0")
+    field(OUT, "$(FULL_PREFIX)_INHIBIT_FRONTBEAMSTOP PP MS")
+}
+
+record(dfanout, "$(FULL_PREFIX)_INHIBIT_FRONTBEAMSTOP") {
+    field(DESC, "Stop and inhibit fbs movement")
+    field(DOL, "$(FULL_PREFIX)INHIBIT_FRONTBEAMSTOP PP MS")
```

Comment:
Why is this input PP ?

Reply:
I think I meant to put this PP against stop, swapped now